rotors_description/urdf/neo11_generic_odometry_sensor.gazebo

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
```

Comment:
I think this file is no longer needed. there is a mav_generic_odometry_sensor.gazebo file for all mavs.